convert.c

```diff
@@ -256,6 +257,117 @@ static int will_convert_lf_to_crlf(size_t len, struct text_stat *stats,
 
 }
 
+static struct encoding {
+	const char *name;
+	iconv_t *to_git;
+	iconv_t *to_worktree;
+	struct encoding *next;
+} *encoding, **encoding_tail;
+static const char *default_encoding = "utf-8";
```

Comment:
According to the [`iconv_open` `man` pages]((http://man7.org/linux/man-pages/man3/iconv_open.3.html)):

> For the GNU C library, the permitted values are listed by the `iconv --list` command …

This list contains `UTF-8` in uppercase but not lowercase. Could that be an issue?

Reply:
AFAIK the encodings are case insensitive.

Reply:
Okay, I wast just wondering.